### PR TITLE
Force print sensitive data parameter (Issue 177) 

### PR DESCRIPTION
**Pull request by bobi591**
Hey @rdagumampan!

Hope this one is good too! :)
If not I will do the needed corrections

### CODE REVIEW

File: yuniql-core/ConfigurationService.cs [hunk 1]
```diff
@@ -151,12 +151,13 @@ public void Validate()
         }
 
         ///<inheritdoc/>
-        public string PrintAsJson(bool redactSensitiveText = true)
+        public string PrintAsJson()
         {
             var _configuration = GetConfiguration();
```
Comment (rdagumampan) [inline yuniql-core/ConfigurationService.cs:156]: As convention, we dont use underscore for local variables. We only use underscore `_` for local private class level variables. Please name is as `configuration` unless it has name conflicts.

File: yuniql-core/FileTraceService.cs [hunk 2]
```diff
@@ -18,6 +18,9 @@ public FileTraceService()
         ///<inheritdoc/>
         public bool IsDebugEnabled { get; set; } = false;
 
+        ///<inheritdoc/>
+        public bool TraceSensitiveData { get; set; } = false;
```
Comment (rdagumampan) [inline yuniql-core/FileTraceService.cs:22]: As convention, we make boolean properties starts with `Is` to show clear intent that its true/false. Please name this as `IsTraceSensitiveData` :)

File: yuniql-cli/Program.cs [hunk 3]
```diff
@@ -74,6 +74,8 @@ private static int Dispatch<T>(Func<T, int> command, T opts, ITraceService trace
             Console.ResetColor();
 
             traceService.IsDebugEnabled = opts.IsDebug;
+            traceService.TraceSensitiveData = opts.TraceSensitiveData;
```
Comment (rdagumampan) [inline yuniql-cli/Program.cs:77]: As convention, we start all boolena properties with `Is` :)

File: yuniql-extensibility/ITraceService.cs [hunk 4]
```diff
@@ -10,6 +10,11 @@ public interface ITraceService
         /// </summary>
         bool IsDebugEnabled { get; set; }
 
+        /// <summary>
+        /// When true, sensitive data is not getting redacted.
+        /// </summary>
+        bool TraceSensitiveData { get; set; }
```
Comment (rdagumampan) [inline yuniql-extensibility/ITraceService.cs:16]: `IsTraceSensitiveData`

**Review by rdagumampan — CHANGES_REQUESTED** [review]
Hi @bobi591, thank you for this great contribution. You really getting the key intent of this issue and there very little to explain. I can also see you started extending the unit tests. Thanks really helpful in ensuring our changes doesn't break existing functions. 

Just minor change, see comments and its all good.

**Comment by rdagumampan** [comment]
Also please comment in this issue so I can assign to you...
https://github.com/rdagumampan/yuniql/issues/177

File: yuniql-core/ConfigurationService.cs (hunk 1 above)
Comment (bobi591) [reply yuniql-core/ConfigurationService.cs]: thanks for the feedback! :) fixed with latest commit

File: yuniql-cli/Program.cs (hunk 3 above)
Comment (bobi591) [reply yuniql-cli/Program.cs]: thanks for the feedback! :) fixed with latest commit

**Pull request merged**